README.md

```diff
@@ -20,6 +20,9 @@ Then to run tests run:
 To just run the application run:
 `make run`
 
+To run the mock api run:
+`make api`
+
```

Comment:
Here's a little more information for the readme about the command line switches added to the mock-api:

```suggestion
To run the mock api run:
`make api`

The mock api also accepts optional command line switches to affect the response:
`npm run mock-api -- --tech-record=[option] --test-result=[option]`

where `[option]` is:
- `NotFound` - returns a 404
- `ServiceError` - returns a 500
```

Reply:
Updated with a nice table